test/e2e/diff_test.go

```diff
@@ -375,3 +375,30 @@ data:
 
 	require.Containsf(t, out, expectedOutput, "Did not find expected diff output")
 }
+
+func TestKappNotCreateConfigmapOnDiffRun(t *testing.T) {
```

Comment:
```suggestion
func TestDiffRun(t *testing.T) {
```
I think we don't have an existing test that asserts _just_ diff run output. Should we also add that to the same test?